sdk/ai/azure-ai-inference/tests/model_inference_test_base.py

```diff
@@ -92,6 +89,16 @@
     azure_ai_image_embeddings_key="00000000000000000000000000000000",
 )
 
+ServicePreparerChatCompletionsWithAudio = functools.partial(
+    EnvironmentVariableLoader,
+    "azure_ai_chat_with_audio",
+    azure_openai_chat_audio_endpoint="https://your-deployment-name.openai.azure.com/openai/deployments/gpt-4o-audio-preview",
```

Comment:
all variables here must start with "azure_ai_chat_with_audio", or whatever you put in line 94